docs/pages/database-access/guides/gui-clients.mdx

```diff
@@ -353,3 +353,75 @@ Click `Add Redis Database`.
 Congratulations! You have just connected to your Redis instance.
 
 ![Redis Insight Connected](../../../img/database-access/guides/redis/redisinsight-connected.png)
+
+
+## Snowflake - JetBrains (IntelliJ, Goland, DataGrip, PyCharm etc.)
+
+<Admonition type="note">
+  Snowflake integration works only in the authenticated proxy mode. Remember to add `--tunnel` flag when starting the proxy, example:
+  ```
+  tsh proxy db --tunnel --port 2000 snowflake
+  ```
+</Admonition>
+
+In "Database Explorer" click and the add button and then pick "Data Source" and pick "Snowflake":
+
+![JetBrains Add Database](../../../img/database-access/guides/snowflake/jetbrains-add-database.png)
+
+Next set "Host" to `localhost` and "Port" as the one returned by the proxy command (`2000` in the example above).
+Set the "Username" to match the one that you are connecting to using Teleport and enter any value in the "Password" field:
+
+![JetBrains General](../../../img/database-access/guides/snowflake/jetbrains-general.png)
+
+Switch to "Advanced" tab and set any value as `account` and add new record named `ssl` with value `off`:
+
+![JetBrains Advanced](../../../img/database-access/guides/snowflake/jetbrains-advanced.png)
+
+<Admonition type="note">
+  Password and account name are being ignored as Teleport uses the values provided in the Database Agent configuration.
+  SSL set to `off` disables only encryption on local machine. Connection to Snowflake is encrypted by Teleport.
+</Admonition>
+
+Now you can click "Test Connection" to check your configuration.
+
+![JetBrains Success](../../../img/database-access/guides/snowflake/jetbrains-success.png)
+
+Congratulations! 🎉 You have just connected to your Snowflake instance.
+
+## Snowflake - DBeaver
+
+<Admonition type="note">
```

Comment:
As per an earlier comment, we should frame this `Admonition` as an instruction that everyone should follow rather than a warning and remove the `Admonition` tags.